torchbenchmark/util/distributed/core_model/trainer.py

```diff
@@ -0,0 +1,137 @@
+from datetime import datetime
```

Comment:
i'm kinda confused why this is a new file, don't we already have a very similar file that does stuff with args for --distributed and sets up the DDP wrapper?

Reply:
Yes, this trainer file is for core model (the model list in https://github.com/pytorch/benchmark/tree/main/torchbenchmark/models), it is very similar to what we did for e2e_models. The difference:
1. we have way more core models than e2e_models, so users can get DDP+torchdynamo performance numbers on all core models
2. since core models only have the `train` function, and don't have the `forward`, `backward`, `optimizer` stages, we are only measuring the latency of the `train` function, instead of these three individual stages